spyder/app/mainwindow.py

```diff
@@ -574,7 +574,8 @@ def setup(self):
                 # Set style proxy to fix combobox popup on mac and qdark
                 qapp = QApplication.instance()
                 qapp.setStyle(self._proxy_style)
-            dark_qss = qdarkstyle.load_stylesheet_from_environment()
+            dark_qss = qdarkstyle.load_stylesheet(
+                        qt_api=os.environ.get('QT_API', None))

```

Comment:
Please remove `None` from `os.environ.get` because that's the default value returned by `os.environ.get` if the environment variable you're passing to it can't be found.

Also, please do the same for all other changes you did in this PR.